framework/src/test/java/org/tron/core/services/http/GetBrokerageServletTest.java

```diff
@@ -0,0 +1,46 @@
+package org.tron.core.services.http;
+
+import static org.junit.Assert.assertEquals;
+
+import com.alibaba.fastjson.JSONObject;
+
+import java.io.UnsupportedEncodingException;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+
+public class GetBrokerageServletTest {
+
+  private MockHttpServletRequest request;
+  private MockHttpServletResponse response;
+
+  @Before
+  public void setUp() {
+    request = new MockHttpServletRequest();
+    request.setMethod("POST");
+    request.setContentType("application/x-www-form-urlencoded");
+    request.setCharacterEncoding("UTF-8");
+
+    response = new MockHttpServletResponse();
+  }
+
+  @Test
+  public void getBrokerageTest() {
+    request.addParameter("address", "");
+    GetRewardServlet getRewardServlet = new GetRewardServlet();
+    getRewardServlet.doPost(request, response);
+    try {
+      String contentAsString = response.getContentAsString();
+      JSONObject result = JSONObject.parseObject(contentAsString);
+      int reward = (int)result.get("reward");
+      Assert.assertEquals(0, reward);
+      String content = (String) result.get("Error");
+      Assert.assertNull(content);
+    } catch (UnsupportedEncodingException e) {
+      e.printStackTrace();
```

Comment:
Assert.fail(); is a good choice

Reply:
I'm not expecting an exception to be thrown; the end result of this use case is a normal return of the

Reply:
Does it mean that even if an UnsupportedEncodingException is thrown, it's still normal?

Reply:
Using a statement when pointing out a problem is a good way to respond; if you use a rhetorical question, I can't be sure of your intentions and it doesn't help to accurately describe the problem.

Reply:
The point of this use case is not to test for the UnsupportedEncodingException, but rather to determine the value of null.

Reply:
`Assert.fail()` is added where it is not possible to reach in the expectation, once it is reached, indicates that the test has failed, the result is not as expected.